articles/azure/azs-how-resize-disk-ps.md

```diff
@@ -0,0 +1,221 @@
+---
+title: How to resize a disk using PowerShell | UKCloud Ltd
```

Comment:
Do we want to make this title consistent with the other resize disks article? That one is called How to resize virtual machine disks using the UKCloud Azure Stack portal. So we could either change this one to How to resize virtual machine disks using PowerShell. Or rename the portal one to How to resize a disk using the UKCloud Azure Stack portal.

Don't mind which option you go for, but would be good for them to be consistent.

Reply:
Fixed here: https://github.com/UKCloud/documentation/pull/542